v2/tools/generator/internal/codegen/testdata/ArmResource/Arm_test_required_and_optional_resource_references_azure.golden

```diff
@@ -298,6 +298,11 @@ type FakeResource_APIVersion_Spec string
 
 const FakeResource_APIVersion_Spec_20200601 = FakeResource_APIVersion_Spec("2020-06-01")
 
+// Mapping from string to FakeResource_APIVersion_Spec
+var fakeResource_APIVersion_Spec_Values = map[string]FakeResource_APIVersion_Spec{
```

Comment:
Any idea why this apiversion in the test isn't getting excluded?

Reply:
Because the type isn't called exactly `APIVersion` - I'm guessing because the name isn't being simplified in the same way as the main pipeline.